pyproject.toml

```diff
@@ -19,6 +19,43 @@ pydantic-settings = "^2.1.0"
 ruff = "^0.1.7"
 black = "^23.11.0"
 pytest = "^7.4.3"
+pytest-cov = "^4.1.0"
+pytest-mock = "^3.12.0"
+pytest-env = "^1.1.3"
+pytest-asyncio = "^0.21.1"
+httpx = "^0.24.1"
+
+
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
+pythonpath = ["src"]
+addopts = [
+  "--cov-report=term-missing",
+  "--cov-branch",
+  "--cov-report=xml",
+  "--cov-report=term",
+  "--cov=src",
+  "-vv",
+  "--strict-markers",
+  "-rfE",
+]
+testpaths = ["tests"]
+env = [
+  "INST_DB_SCHEMA=main",
+  "INST_DB_USER=user",
+  "INST_DB_PWD=user",
+  "INST_DB_HOST=localhost:5432",
+  "INST_DB_NAME=fi",
```

Comment:
Shouldn't need the institution DB env vars in this repo.

Reply:
I forgot to remove those.